GitPrompt.ps1

```diff
@@ -82,6 +82,9 @@ $global:GitPromptSettings = New-Object PSObject -Property @{
 
     AutoRefreshIndex                            = $true
 
+    # Valid values are "Full", "Compact", and "Minimal"

```

Comment:
For 1.0 we could consider converting `$GitPromptSettings` to a giant config function of doom, as @jaykul proposed [long ago](https://github.com/dahlbyk/posh-git/pull/111).